frontend/fingertips-frontend/mock/server/handlers.ts

```diff
@@ -15,10 +19,108 @@ export const handlers = [
   http.get(`${baseURL}/WeatherForecast`, async () => {
     const resultArray = [[getGetWeatherForecast200Response(), { status: 200 }]];
 
+    return HttpResponse.json(...resultArray[next() % resultArray.length]);
+  }),
+  http.get(`${baseURL}/indicators`, async () => {
+    const resultArray = [
+      [getFilterIndicators200Response(), { status: 200 }],
+      [getFilterIndicators400Response(), { status: 400 }],
+      [getFilterIndicators500Response(), { status: 500 }],
+    ];
+
+    return HttpResponse.json(...resultArray[next() % resultArray.length]);
+  }),
+  http.get(`${baseURL}/indicators/:indicatorId`, async () => {
+    const resultArray = [
+      [getGetIndicator200Response(), { status: 200 }],
+      [getGetIndicator404Response(), { status: 404 }],
+      [getGetIndicator500Response(), { status: 500 }],
+    ];
```

Comment:
Again, I think it's simpler for now if we just return a successful response.
```suggestion
    const resultArray = [
      [getGetIndicator200Response(), { status: 200 }]
    ];
```